packages/contracts/deployments/README.md

```diff
@@ -125,6 +118,7 @@ Network : __goerli (chain id: 5)__
 | OVM_StateManagerFactory               | [0x24C7F0a4a2B926613B31c4cDDA4c0f90c0772f2b](https://goerli.etherscan.io/address/0x24C7F0a4a2B926613B31c4cDDA4c0f90c0772f2b) |
 | OVM_StateTransitionerFactory          | [0x703303Ce2d92Ef95F17a622E3d538390251165E8](https://goerli.etherscan.io/address/0x703303Ce2d92Ef95F17a622E3d538390251165E8) |
 | Proxy__OVM_L1CrossDomainMessenger     | [0xa85716330ff84Ab312D5B43F3BfDcC7E650fd88A](https://goerli.etherscan.io/address/0xa85716330ff84Ab312D5B43F3BfDcC7E650fd88A) |
+| Proxy__OVM_L1ETHGateway               | [0xA721CF3e39E5cB4CfEEc0e32EE05B3D05AA9aE39](https://goerli.etherscan.io/address/0xA721CF3e39E5cB4CfEEc0e32EE05B3D05AA9aE39) |
```

Comment:
This also changed when I ran the script. 
It looks like the [artifact](https://github.com/ethereum-optimism/optimism/pull/603/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-5b6fb2aa87259920aaf22a127f8dcfd4064c87f1093aa888ae54357194a02977) was added in #603, but the README [wasn't updated](https://github.com/ethereum-optimism/optimism/pull/603/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-29c13e9d05968171f1b77293799876fe6afc393f674123ee883d49c50f25a14e) in the PR.